src/LodeRunner.Core/SystemConstants.cs

```diff
@@ -133,5 +133,15 @@ public class SystemConstants
         /// The default API web host port.
         /// </summary>
         public const int DefaultApiWebHostPort = 8080;
+
+        /// <summary>
+        /// The integration clients URI path.
+        /// </summary>
+        public const string IntegrationClientsUri = "/api/Clients";
```

Comment:
I likely missed it, but is the IntegrationClientsUri being used?

Reply:
Removed reference and constant, and updated code to use an existing constant under LodeRunner.API.SystemConstants